server/routerlicious/packages/routerlicious-base/src/alfred/app.ts

```diff
@@ -52,6 +53,18 @@ export function create(
     // Express app configuration
     const app: express.Express = express();
 
+    // initialize RestLess server translation
+    const restLessMiddleware: () => express.RequestHandler = () => {
+        const restLessServer = new RestLessServer();
+        return (req, res, next) => {
+            restLessServer
+                .translate(req)
+                .then(() => next())
+                .catch(next);
+        };
+    };
+    app.use(restLessMiddleware());
```

Comment:
This is okay for now but we should create a work item to move this middleware to a shared library and have r11s and historian use it.

Reply:
Also how do you know the client is using restless as well? I see you have the following conditions in RestlessServer:

1. Method should be post
2. request.headers["content-type"]?.toLowerCase().includes("application/x-www-form-urlencoded")

Does these two conditions guarantee that calls are coming from restless client?

Reply:
In the case of FluidFramework, and specifically our driver, yes. This wouldn't be a complete solution if this were a general purpose library. One way could be to add a form property indicating that it is restless, so then it would be certain.

Reply:
I was thinking about adding a new property as well. But that can wait